components/Schema.js

```diff
@@ -57,7 +57,7 @@ function SchemaProp({ prop, propName, required = false, path = '', description =
     }
   });
 
-  const additionalProperties = prop.additionalProperties() && prop.additionalProperties().properties()
+  const additionalProperties = prop.additionalProperties() && typeof prop.additionalProperties() !== "boolean" && prop.additionalProperties().properties()
```

Comment:
Beside `boolean` and `object`, what other types `prop.additionalProperties()` could be here?

Instead of doing a negative type check, I would opt for checking if the type is `object`.

Reply:
Only those two types, so we can do that yea 👍

Reply:
Just to clarify, It is not because of styling but readability. 👍

Reply:
Yea, I usually just jump on the first thought that fixes the issue 😅 It is a good point